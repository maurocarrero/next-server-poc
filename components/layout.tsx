import globalStyles from '../styles/global.js';
import { node } from 'prop-types';

const Layout = ({ children }) => (
  <div className="page-layout">
    {children}
    <style jsx global>
      {globalStyles}
    </style>
  </div>
);

Layout.propTypes = {
  children: node.isRequired
};

export { Layout };
