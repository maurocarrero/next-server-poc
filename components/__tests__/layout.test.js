import React from 'react';
import { render } from '@testing-library/react';
import { Layout } from '../layout';

it('should match', () => {
  const { container } = render(<Layout><h1>{`Hi!`}</h1></Layout>);
  expect(container).toMatchSnapshot();
});
