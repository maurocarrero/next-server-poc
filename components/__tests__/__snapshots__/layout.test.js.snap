// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match 1`] = `
<div>
  <div
    class="page-layout"
  >
    <h1>
      Hi!
    </h1>
  </div>
</div>
`;
